packages/ckeditor5-code-block/src/indentcodeblockcommand.ts

```diff
@@ -16,27 +15,23 @@ import {
 
 /**
  * The code block indentation increase command plugin.
- *
- * @extends module:core/command~Command
  */
 export default class IndentCodeBlockCommand extends Command {
-	constructor( editor ) {
+	/**
+	 * A sequence of characters added to the line when the command is executed.
+	 */
+	private _indentSequence: string;
+
+	constructor( editor: Editor ) {
 		super( editor );
 
-		/**
-		 * A sequence of characters added to the line when the command is executed.
-		 *
-		 * @readonly
-		 * @private
-		 * @member {String}
-		 */
-		this._indentSequence = editor.config.get( 'codeBlock.indentSequence' );
+		this._indentSequence = editor.config.get( 'codeBlock.indentSequence' ) as string;
```

Comment:
```suggestion
		this._indentSequence = editor.config.get( 'codeBlock.indentSequence' )!;
```

This type should be inferred from augmented `EditorConfig`.